src/InSAR/InSAR.jl

```diff
@@ -16,7 +16,7 @@ _doppler_centroid_frequency(x, dc_param, x0) = dc_param[1] .+ dc_param[2].*(x .-
             dc_tau_0::Float64, fm_coefficient::Vector{Float64},
             fm_tau_0::Float64, f_c::Float64, lines_per_burst::Int64,
             number_of_samples::Int64, delta_t_s::Float64,
-            delta_tau_s::Float64, tau_0::Number, c=LIGHT_SPEED::Real) where T <: Integer
+            delta_tau_s::Float64, tau_0::Number, c=LIGHT_SPEED::Real) where T <: Number
```

Comment:
why is Number better than Integer?

Reply:
We need to compute the phase ramp for non-interger locations eg. row 1.23 line 4.23 when re-ramping the image. We need to reramp the image at the resampled locations of the secondary image.

I will actually change `Number` to `Real`. Rows and samples should not be complex but only real numbers.